test/tools/libtesteth/TestOutputHelper.h

```diff
@@ -32,6 +32,7 @@ class TestOutputHelper
 public:
 	TestOutputHelper(size_t _maxTests = 1) { TestOutputHelper::initTest(_maxTests); }
 	static bool passTest(std::string const& _testName);
+	static void showProgress();
```

Comment:
Document that this function has to be called on every test.  Currently, the name reads like just a printing function that can be skipped if the console is too crowded.